ext/gd/gd.c

```diff
@@ -351,13 +353,68 @@ void php_gd_error_method(int type, const char *format, va_list args)
 }
 /* }}} */
 
+static zend_object_handlers gd_ext_image_object_handlers;
+
+static const zend_function_entry gd_image_methods[] = {
+        PHP_FE_END
+};
+
+static gd_ext_image_object* gd_ext_image_object_from_ptr(zend_object* obj) {
+    return (gd_ext_image_object *)((char *)(obj) - XtOffsetOf(gd_ext_image_object, std));
+}
+
+static zend_object* gd_ext_image_object_create(zend_class_entry * class_type) {
+    gd_ext_image_object* intern = emalloc(sizeof(gd_ext_image_object));
```

Comment:
need to allocate space for inline properties (and implicitly guards) for ext compatibility ...